<launch>
	<!-- Vehicle Parameters -->
	<param name="vehicle_id" type="int" value="1" />
	<param name="vehicle_name" type="str" value="uav1" />
	<param name="vehicle_type" type="int" value="4" />
    <param name="net_num" type="int" value="1"/>
        <param name="net_1" type="int" value="50"/>
    
	<!-- Ground Station Parameters -->
	<param name="gs_id" type="int" value="1234" />
	<param name="gs_channel" type="int" value="1" />
	<param name="gs_lat" type="double" value="38.835675" />
	<param name="gs_long" type="double" value="-9.377597" />
	<param name="gs_alt" type="double" value="85" />
	<param name="gs_nettype" type="int" value="40" />


	<node name="piccolo_driver_1" pkg="piccolo_ros" type="piccolo_driver"
		output="screen">
		<param name="host" type="str" value="62.28.239.26" />
		<param name="port" type="str" value="2001" />
	</node>
	<node name="autopilot_interface_node_1" pkg="ros_vehicle"
		type="autopilot_interface_node" output="screen">
		<!-- refresh wp interval in seconds -->
		<param name="refresh_wps_interval" type="int" value="60" />

	</node>
	<node name="mission_manager_node_1" pkg="ros_vehicle" type="mission_manager_node"
		output="screen">
		<param name="waypoint_move_id" type="int" value="99" />
	</node>
	<node name="mse_processor_node_1" pkg="ros_vehicle" type="mse_processor_node"
		args="1" output="screen">
		
	</node>
	<node name="task_manager_node_1" pkg="ros_vehicle" type="task_manager_node"
		args="1" output="screen" />
	<node name="mse_gateway_node_1" pkg="ros_vehicle" type="mse_gateway_node"
		args="1" output="screen">
		<param name="port" type="int" value="3555" />
		<param name="num_masters" type="int" value="1" />
		<param name="master_host_1" type="str" value="127.0.0.1" />
		<param name="master_port_1" type="int" value="4555" />
	</node>
	<node name="ais_manager_node_1" pkg="ros_vehicle" type="ais_manager_node"
		output="screen">
		<param name="ais_network" type="int" value="50" />
		<param name="ais_vehicle_type" type="int" value="3" />
	</node>

</launch>



